ui/lib/core/addon/components/box-radio.js

```diff
@@ -16,11 +16,18 @@
  * @param {string} [tooltipMessage=default] - The message that shows in the tooltip if the radio option is disabled
  */
 
-import Component from '@ember/component';
+import Component from '@glimmer/component';
 import layout from '../templates/components/box-radio';
+import { setComponentTemplate } from '@ember/component';
 
-export default Component.extend({
-  layout,
-  disabled: false,
-  tooltipMessage: 'This option is not available to you at this time.',
-});
+class BoxRadio extends Component {
+  disabled = false;
+  tooltipMessage = 'This option is not available to you at this time.';
+  onRadioChange = '';
+
+  handleClick(method, value) {
```

Comment:
Continuing my comment from the hbs file, I was initially very confused what onRadioChange was for, and I think simplifying `handleClick` to only take value, and then call `this.onRadioChange` may help. Or, we could skip `handleClick` alltogether and do:

```js
{{on "click" (fn @onRadioChange @type)}}
```
on the template, which eliminates the need for handleClick altogether, especially since there are no side effects we need to handle within the component. This would potentially require onRadioChange being set to a sane default...or maybe it's ok to fail if it's not passed in because not passing it kind of defeats the purpose of this component. 

Sorry for the long rambling thoughts, let me know if any of this doesn't make sense.